extensions/EXT/EXT_framebuffer_blit_layers.txt

```diff
@@ -0,0 +1,107 @@
+Name
+
+    EXT_framebuffer_blit_layers
+
+Name Strings
+
+    GL_EXT_framebuffer_blit_layers
+
+Contacts
+
+    Simon Zeni, Status Holdings Ltd. <simon@zeni.ca>
+
+Contributors
+
+    Simon Zeni
+
+Status
+
+    Draft
+
+Version
+
+    Version 0.1, 2022/09/16
+
+Number
+
+    OpenGL Extension #558
+    OpenGL ES Extension #341
+
+Dependencies
+
+    OpenGL ES 3.0 is required
+
+    This extension is written against the OpenGL ES 3.2 specification
+
+Overview
+
+    This extensions defines the behaviour for copying data from one layered
+    framebuffer to another layered framebuffer.
+
+New Procedures and Functions
+
+    void BlitFramebufferLayersEXT(int srcX0, int srcY0, int srcX1, int srcY1,
+                                  int dstX0, int dstY0, int dstX1, int dstY1,
+                                  bitfield mask, enum filter);
+
+    void BlitFramebufferLayerEXT(int srcX0, int srcY0, int srcX1, int srcY1,
+                                 int srcLayer, int dstX0, int dstY0, int dstX1,
+                                 int dstY1, int dstLayer, bitfield mask,
+                                 enum filter);
+
+New Tokens
+
+    None
+
+Additions to Chapter 4 of the OpenGL 3.0 Specification (Per Fragment Operations
```

Comment:
This should also mention where in the OpenGL spec this addition is made. I guess OpenGL 4.6 section 18.3.1.

Reply:
Shouldnt it be OpenGL 3? It introduces `glBlitFramebuffer`, same as ES 3.0

Reply:
The extension can apply edits to any version of the core specs, it's whatever version is mentioned in the Dependencies section. In general extensions usually apply edits to the latest versions of the core specs. The functionality can still apply to earlier versions. So in this case the Dependencies section might say:
```
Dependencies

OpenGL 3.0, OpenGL ES 3.0, or GL_EXT_framebuffer_blit required

This extension is written against the OpenGL 4.6 Core and OpenGL ES 3.2 specifications.
```

Reply:
Thank you for the answer, I'll finish the edits today